Libraries/Components/Picker/PickerIOS.macos.js

```diff
@@ -25,6 +25,19 @@ var processColor = require('processColor');
 var createReactClass = require('create-react-class');
 var requireNativeComponent = require('requireNativeComponent');
 
+import type {ColorValue} from 'StyleSheetTypes';
```

Comment:
So changes to this file look like they're coming from fb's PickerIOS.ios.js. I'm not saying that's right/wrong for Mac, but are you purposefully keeping the mac version in parity with the iOS version? This doesn't look like what we have in sdx-platform is why I bring it up